tests/unit/test_cli.py

```diff
@@ -155,6 +155,35 @@ def test_set_stdin_yaml(cli_runner, vault_with_token):
     assert vault_with_token.db == {"a": {"b": "c"}}
 
 
+def test_set_prompt_value(cli_runner, vault_with_token):
+
+    result = cli_runner.invoke(cli.cli, ["set", "--prompt", "a", "b"])
+
+    assert result.exit_code != 0
+    assert vault_with_token.db == {}
+
+
+def test_set_prompt_value_stdin(cli_runner, vault_with_token):
```

Comment:
```suggestion
def test_set_with_both_prompt_and_stdin(cli_runner, vault_with_token):
```
(I belive the `value`in the test name was a mistake, if I understand roeprly the previous test name)